contracts/GovernanceUtils.sol

```diff
@@ -12,7 +12,7 @@
 
 // SPDX-License-Identifier: MIT
 
-pragma solidity <0.9.0;
+pragma solidity 0.8.4;
```

Comment:
Why we don't do semver `^0.8.4`?

Reply:
We do not want to stay on the most common one, per the recommendation of auditors.